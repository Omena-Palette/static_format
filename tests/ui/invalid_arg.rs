use static_format::const_format;

fn hello() -> u8 {
    5
}

fn main() {
    const_format!("{}", hello());
}