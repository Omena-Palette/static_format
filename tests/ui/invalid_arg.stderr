error: All arguments must be or expand into literals. (Same constraints as `concat!`)
 --> tests/ui/invalid_arg.rs:8:25
  |
8 |     const_format!("{}", hello());
  |                         ^^^^^
